details/kibana-7.8.x-and-older.md

```diff
@@ -811,7 +811,10 @@ Let's start to add to our access control list (found in $ES\_PATH\_CONF/config/r
 
 ```
 readonlyrest:
-  audit_collector: true
+  audit:
+    enabled: true
+    outputs:
```

Comment:
`outputs: [index]`

I wonder if we shouldn't assume that `outputs: [index]` is a default setting? (cc @sscarduzio)
So, the minimal configuration could look like this:

```
audit:
  enabled: true
```

or

```
audit.enabled: true
```

We should notice that previous minimal configuration was `audit_collector: true`

Reply:
empty or absent outputs array should imply outputs=index, and +1 for the backward compat.

Reply:
AFAIR the empty array is an invalid state and IMO it's better to leave it like this.

Reply:
Yeah makes sense!

Reply:
Fixed